src/ArcCommander/CLIArguments/AssayArgs.fs

```diff
@@ -67,8 +68,8 @@ type AssayUpdateArgs =
 
 /// CLI arguments for interactively editing existing assay metadata.
 type AssayEditArgs = 
-    | [<AltCommandLine("-s")>][<Unique>] StudyIdentifier of study_identifier : string
-    | [<Mandatory>][<AltCommandLine("-a")>][<Unique>] AssayIdentifier of assay_identifier : string
+    | [<AltCommandLine("-s")>][<Unique>]                StudyIdentifier of study_identifier : string
+    | [<Mandatory>][<AltCommandLine("-a")>][<Unique>]   AssayIdentifier of assay_identifier : string
```

Comment:
Why no "FileName" Attribute here?

Reply:
You are correct here. But not all AssayIdentifier and StudyIdentifier arguments should get the `FileName` attribute. Only those that create/change the filename.